webpack/assets/javascripts/react_app/components/common/DateTimePicker/DateTimePicker.js

```diff
@@ -14,24 +14,21 @@ import { MONTH } from './DateComponents/DateConstants';
 import './date-time-picker.scss';
 
 class DateTimePicker extends React.Component {
-  state = {
-    value: new Date(this.props.value),
-    typeOfDateInput: MONTH,
-    isTimeTableOpen: false,
-    left: 'auto',
-  };
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: new Date(this.props.value),
+      typeOfDateInput: MONTH,
+      isTimeTableOpen: false,
+      hiddenValue: this.props.hiddenValue,
+    };
+  }
+
   formatDate = () => {
-    const { locale } = this.props;
     const { value } = this.state;
-    const options = [
-      { year: 'numeric', month: 'numeric', day: 'numeric' },
-      { hour: '2-digit', minute: '2-digit' },
-    ];
-    return `${value.toLocaleString(locale, options[0])} ${value.toLocaleString(
-      locale,
-      options[1]
-    )}`;
+    return `${value.getFullYear()}-${value.getMonth()}-${value.getDate()} ${value.getHours()}:${value.getMinutes()}:00`;
```

Comment:
didn't found a date formator (except using a 3rd party package, which feels redundant)

Reply:
I am ok with this solution, I would adress it in a follow up, as I believe there is going to be work on DateTimePicker in patternfly-react, there i would expect ability to define both formats (display and value) through props.

Reply:
Just two nits:

- `getMonth()` is returning month starting with `0` so January is 0 and so on, what doesn't feel right, I guess we would get it parsed wrong on the server
- I am missing the alignment by zero, so it just looks weird to me, but it shouldn't be a problem.

Reply:
it would be great to fix these two things, then we can go ahead and merge

Reply:
done :)